server/sonar-db-dao/src/main/java/org/sonar/db/webhook/WebhookDeliveryLiteDto.java

```diff
@@ -28,8 +28,10 @@
   protected String uuid;
   /** Component UUID, can't be null */
   protected String componentUuid;
-  /** Compute Engine task UUID, can't be null */
+  /** Compute Engine task UUID, can be null */
   protected String ceTaskUuid;
+  /** Compute Engine analysis UUID, can't be null */
```

Comment:
"Compute Engine analysis UUID" <-- "Compute Engine" is too much

Reply:
actually, analysis_uuid can be null, when webhook is called on a very early failure during the processing of a report